kubernetes-webdemo.yaml

```diff
@@ -18,6 +18,27 @@ spec:
           # See
           # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
           # for documentation on the resources section.
+          env:
+            - name: DEMO_POSTGRES_HOST
```

Comment:
How do you enter these secrets into Kubernetes?  We should probably document this--although I'm fine starting with documenting the web demo in general.

Reply:
I haven't tried yet, but when I do I'm going to follow this:

https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually

Reply:
ok, I did this, check out

```
kubectl get secret allennlp-demo-postgres --namespace allennlp -o yaml
```